spec/dummy/app/models/invoice.rb

```diff
@@ -92,4 +92,18 @@ def self.colors
   def set_default_state
     self.state ||= 'pending'
   end
+
+  def self.ransackable_attributes(auth_object = nil)
```

Comment:
⚠️ **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Lint/UnusedMethodArgument: Unused method argument - auth_object. If it's necessary, use _ or _auth_object as an argument name to indicate that it won't be used. If it's unnecessary, remove it. You can also write as ransackable_attributes(*) if you want the method to accept any arguments but don't care about them.